valkey.conf

```diff
@@ -1809,6 +1815,7 @@ aof-timestamp-enabled no
 # Example:
 #
 # cluster-announce-ip 10.1.1.5
+# cluster-announce-alt-ip ::ffff:a01:105
```

Comment:
Did I miss sth? I do not find where we use it.

Reply:
My mistake. This was from an earlier idea.

I'm changing this to an example of IPv6, like

    cluster-announce-client-ipv6 2001:db8::8a2e:370:7334